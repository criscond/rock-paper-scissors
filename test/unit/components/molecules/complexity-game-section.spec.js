import gameComplexityOptions from '../../../../src/js/components/molecules/complexity-game-section';
describe('Complexity Game Section', () => {

	it('should create a section', () => {
		expect(gameComplexityOptions()).toMatchSnapshot();
	});
});
