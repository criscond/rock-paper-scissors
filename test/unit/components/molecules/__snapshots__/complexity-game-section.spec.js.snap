// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Complexity Game Section should create a section 1`] = `
<section>
  <span
    class=""
  >
    Game complexity: 
  </span>
  <select
    id="gameComplexity"
  >
    <option
      value="1"
    >
      Three weapons
    </option>
    <option
      value="2"
    >
      Five weapons
    </option>
  </select>
</section>
`;
